src/app/styles/index.scss

```diff
@@ -12,4 +12,82 @@
 @font-face {
     font-family: Rubik;
     src: url(fonts/Rubik-Regular.ttf);
+}
+
+@layer components {
+   .h1 {
+      @apply font-rubik font-normal text-8xl
+   }
+
+   .h2 {
+      @apply font-rubik font-semibold text-[50px]
+   }
+
+   .h3 {
+      @apply font-rubik font-medium text-5xl
+   }
+
+   .h4 {
+      @apply font-rubik font-normal text-[46px]
+   }
+
+   .h5 {
+      @apply font-rubik font-normal text-4xl
+   }
+
+   .h6 {
+      @apply font-rubik font-light text-2xl
+   }
+
+   .primary {
+      @apply font-rubik font-normal text-xl
+   }
+
+   .primary-bold {
+      @apply font-rubik font-semibold text-[22px]
+   }
+
+   .secondary {
+      @apply font-rubik font-light text-lg
+   }
+
+   .secondary-bold {
+      @apply font-rubik font-medium text-lg
+   }
+
+   .additional {
+      @apply font-rubik font-normal text-base
+   }
+}
+
+.catalog-icon:hover circle {
+    @apply fill-accent-turkus;
+}
+
+.present-icon {
+&:hover {
+    path:nth-child(1){
+        @apply fill-[#4495ff];
+    }
+
+    path:nth-child(3) {
+        @apply fill-accent-turkus;
+    }        
+    
+    path:nth-child(2), path:nth-child(5){
+        @apply fill-[#006eff];
```

Comment:
Please, use custom colors from tailwind.config.js.
If u search this color u will see it on the path in .config: theme/extend/colors/link:'#006EFF',